src/src/Helpers/StringParser.cpp

```diff
@@ -26,6 +26,24 @@
 /********************************************************************************************\
    Parse string template
  \*********************************************************************************************/
+bool hasEscapedCharacter(String& str, const char EscapeChar)
+{
+  String EscStr = F("\\");
+  EscStr += EscapeChar;
+  return (str.indexOf(EscStr)>=0);
+}
+
+void stripEscapeCharacters(String& str)
+{
+  str.replace(F("\\%"), "%"); // Allow system vars to be passed in by using \% instead of %
+  str.replace(F("\\["), "["); // Allow task values to be passed in by using \[ instead of [
+  str.replace(F("\\]"), "]"); // Allow task values to be passed in by using \] instead of ]
+  str.replace(F("\\{"), "{"); // Allow commands to be passed in by using \{ instead of {
+  str.replace(F("\\}"), "}"); // Allow commands to be passed in by using \} instead of }
+  str.replace(F("\\("), "("); // Allow commands to be passed in by using \( instead of (
+  str.replace(F("\\)"), ")"); // Allow commands to be passed in by using \) instead of )

```

Comment:
The replacement strings should also be using the `F()` macro.

Using an array of chars and a `for` loop is even more compact, something like this:
``` c++
  const uint8_t braces[]     = { '%', '[', ']', '{', '}', '(', ')' };
  constexpr uint8_t nrbraces = NR_ELEMENTS(braces);

  for (uint8_t i = 0; i < nrbraces; ++i) {
    str.replace(strformat(F("\\%c"), braces[i]), strformat(F("%c"), braces[i]));
  }
```
On ESP32 saves ca. 50 bytes, on ESP8266 saves ca. 40 bytes in bin size.

Reply:
Why not simply do this:
```c++
    str.replace(strformat(F("\\%c"), braces[i]), String(braces[i]));
```

or:
```c++
    str.replace(concat(F("\\"), braces[i]), String(braces[i]));
```

Reply:
Done!